sevntu-checks/pom.xml

```diff
@@ -16,7 +16,7 @@
   <properties>
     <project.build.sourceEncoding>iso-8859-1</project.build.sourceEncoding>
     <!-- It is compile dependency to checkstyle, this version has to be the same as eclipse-cs depends on -->
-    <checkstyle.eclipse-cs.version>9.1</checkstyle.eclipse-cs.version>
+    <checkstyle.eclipse-cs.version>9.2</checkstyle.eclipse-cs.version>
```

Comment:
Comment about warn that eclipscs version should be same.
We need to wait for eclipscs to upgrade to 9.2 first

Reply:
Added note to PR description.

Reply:
we already on 10.0
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/92711399e264a4ed78e0233f823fd6288f14eb6d/sevntu-checks/pom.xml#L19

this PR is stale